spec/features/user_can_read_email_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+require "pry"
```

Comment:
I don't think you need to require `pry` here. It is required by default by the rails helper